sycl/doc/extensions/LevelZeroBackend/LevelZeroBackend.md

```diff
@@ -0,0 +1,119 @@
+﻿# Level-Zero backend specification
+
+## 1. Introduction
+
+This extension introduces Level-Zero backend for SYCL.
+It is built on top of Level-Zero runtime enabled with [Level-Zero API](https://spec.oneapi.com/level-zero/latest/index.html).
+The Level-Zero backend is aimed to provide the best possible performance of SYCL application on a variety of targets supported.
+The currently suppported targets are all Intel GPUs starting with Gen9.
+
+NOTE: This specification is a draft. While describing the currently implemented behaviors it is known to be not complete nor exhaustive.
+      We shall continue to add more information, e.g. explain general mapping of SYCL programming model to Level-Zero API.
+      It will also be gradually changing to a SYCL-2020 conforming implementation.
+
+## 2. Prerequisites
+
+The Level-Zero loader and drivers need to be installed on the system for SYCL runtime to recognize and enable the Level-Zero backend.
+For further details see <https://software.intel.com/content/www/us/en/develop/articles/intel-oneapi-dpcpp-system-requirements.html>.
+                  
+## 3. User-visible Level-Zero backend selection (and the default backend)
+
+The Level-Zero backend is added to the cl::sycl::backend enumeration:
+
+``` C++
+enum class backend {
+  // ...
+  level_zero,
+  // ...
+};
+```
+
+There are multiple ways in which the Level-Zero backend can be selected by the user.
+        
+### 3.1 Through an environment variable
+        
+The SYCL_DEVICE_FILTER environment variable limits the SYCL runtime to use only a subset of the system's devices.
+By using ```level_zero``` for backend in SYCL_DEVICE_FILTER you can select the use of Level-Zero as a SYCL backend.
+For further details see here: [https://github.com/intel/llvm/blob/sycl/sycl/doc/EnvironmentVariables.md](https://github.com/intel/llvm/blob/sycl/sycl/doc/EnvironmentVariables.md).

```

Comment:
FWIW, you could avoid URL duplication here and elsewhere in this document by again using the `<>` syntax.

Reply:
Fixed